[2022-11-16 01:25:37,515] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2022-01-07T00:00:00+00:00 [queued]>
[2022-11-16 01:25:38,629] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2022-01-07T00:00:00+00:00 [queued]>
[2022-11-16 01:25:38,673] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 01:25:38,687] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-11-16 01:25:38,693] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 01:25:41,858] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): hello_task> on 2022-01-07T00:00:00+00:00
[2022-11-16 01:25:42,297] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_world', 'hello_task', '2022-01-07T00:00:00+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/index.py', '--cfg-path', '/tmp/tmpthbmthfy', '--error-file', '/tmp/tmp6zc8ou6v']
[2022-11-16 01:25:42,710] {standard_task_runner.py:77} INFO - Job 8: Subtask hello_task
[2022-11-16 01:25:41,989] {standard_task_runner.py:52} INFO - Started process 117 to run task
[2022-11-16 01:25:47,987] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.hello_task 2022-01-07T00:00:00+00:00 [running]> on host 001906ffd7c9
[2022-11-16 01:25:55,333] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=hello_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-07T00:00:00+00:00
[2022-11-16 01:25:55,772] {logging_mixin.py:104} INFO - hi everyone
[2022-11-16 01:25:55,789] {python.py:118} INFO - Done. Returned value was: hello world
[2022-11-16 01:25:58,684] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=hello_task, execution_date=20220107T000000, start_date=20221116T012537, end_date=20221116T012558
[2022-11-16 01:26:02,008] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-11-16 01:26:02,150] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 117
[2022-11-16 01:26:02,273] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-11-16 01:26:03,255] {process_utils.py:66} INFO - Process psutil.Process(pid=117, status='terminated', exitcode=1, started='01:25:41') (117) terminated with exit code 1
